rest-service/manager_rest/manager_exceptions.py

```diff
@@ -122,6 +122,11 @@ class NoTokenGeneratorError(ManagerException):
     status_code = 401
 
 
+class InvalidTokenFormatError(ManagerException):
```

Comment:
Can we maybe call this something like InvalidFernetFormatError to differentiate from tokens used for auth?